src/js/views/patients/worklist/worklist_views.js

```diff
@@ -276,7 +276,7 @@ const sortDueOptions = [
       const dueA = a.model.get('due_date');
       const dueB = b.model.get('due_date');
       if (dueA === dueB) {
-        return alphaSort('asc', a.model.get('due_time'), b.model.get('due_time'));
+        return alphaSort('asc', a.model.get('due_time'), b.model.get('due_time'), '24');
```

Comment:
When sorting the row for the `Due: Oldest - Newest` option, this forces actions with a `due_date` to be on top of actions without one (`due_date: null`).

The `'24'` is not needed to achieve this for the `Due: Newest - Oldest` sort option.